ui/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "classnames": "^2.2.6",
```

Comment:
The `ˆ` should not be used since it can install a major version which can impact the project with break changes

Reply:
See my answer ^